### PR TITLE
BTFSINFRA-520 up version number and change uri in api path

### PR DESCRIPTION
**Pull request by richlee33**
unit tests passed  ;)

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -4,6 +4,6 @@ package ipfs
 var CurrentCommit string
 
 // CurrentVersionNumber is the current application's version literal
-const CurrentVersionNumber = "0.1.0"
+const CurrentVersionNumber = "0.2.0"
 
-const ApiVersion = "/go-ipfs/" + CurrentVersionNumber + "/"
+const ApiVersion = "/go-btfs/" + CurrentVersionNumber + "/"
```
Comment (jin-tron) [inline version.go]: There's unit test for this,
core/corehttp/option_test.go:23:		return fmt.Sprintf("%s (%s != %s)\n", errAPIVersionMismatch, version.ApiVersion, tc.userAgent)
core/corehttp/option_test.go:33:		{version.ApiVersion, APIPath + "/test", true, "check!", http.StatusOK},
Please check if unit test can pass after your change.

Comment (richlee33) [reply version.go]: i checked locally and it passed.  it also passed in the linked jenkins unit test job which gates merging.

**Comment by richlee33** [comment]
Jin and i discussed, we will not change the Api setting at this point in time to minimize breaking stuff.

**Comment by richlee33** [comment]
jenkins unit tests passed.  https://builds.bittorrent.com/job/btfs-dev-test/47/  merging.

**Pull request merged**